src/server.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+
+const http = require('http');
+const path = require('path');
+const fs = require('fs');
+const zlib = require('zlib');
+const multiparty = require('multiparty');
+
+const server = http.createServer((req, res) => {
+  if (req.url === '/') {
+    const pathFileIndexHTML = path.join(__dirname, 'index.html');
+
+    const readStream = fs.createReadStream(pathFileIndexHTML);
+
+    res.statusCode = 200;
+    res.setHeader('Content-Type', 'text/html');
+
+    readStream.pipe(res);
+
+    readStream.on('error', (err) => {
+      res.statusCode = 404;
+      res.end(err);
+    });
+
+    return;
+  }
+
+  if (req.url === '/compress') {
+    const form = new multiparty.Form();
+
+    form.parse(req, (err, fields, files) => {
+      if (err) {
+        res.statusCode = 404;
+        res.end('Error');
```

Comment:
Now the `Error` message is uninformative and does not tell the user what exactly went wrong.